package.json

```
@@ -27,23 +27,22 @@
     "web-ledger-client": "^3.1.0"
   },
   "peerDependencies": {
-    "bedrock": "^1.16.0",
-    "bedrock-account": "^2.1.1",
-    "bedrock-express": "^2.0.8",
-    "bedrock-https-agent": "^1.0.1",
-    "bedrock-identity": "^6.1.0",
+    "bedrock": "^1.x - 3.x",
```

Not sure we even need 1 or 2, but if so, fix syntax:
```suggestion
    "bedrock": "1.x - 3.x",
```